packages/shared/src/index.ts

```diff
@@ -1,4 +1,4 @@
-/** The following files are not exported on purpose:
+1; /** The following files are not exported on purpose:
```

Comment:
```suggestion
/** The following files are not exported on purpose:
```

Reply:
Oh good catch! 
I'll commit the fix when the tests are done so we don't have to wait for them to run again, thanks!